builders/ethereum/dev-env/hardhat.md

```diff
@@ -52,13 +52,29 @@ You will need to create a Hardhat project if you don't already have one. You can
 4. Create a Hardhat project
 
     ```sh
-    npx hardhat init
+    npx hardhat --init
```

Comment:
When I ran this with the "--init" it gave me an error "Error HH304: Unrecognised command line argument --init." 

I took the "--" back out and it worked perfectly. May need to discard this change.

Reply:
The latest version of Hardhat (npm install hardhat@3.0.0-next.5) requires it as `--init`

Reply:
So I'm torn here. Should we replace the command for 3. Install Hardhat under "Create a Hardhat Project" to use the latest version instead of just `npm install hardhat`? 

If I follow your instructions, I'm running `npm install hardhat` and the next thing is an init command that won't work with that installation.  The Note with the install command for the version that works the way your guide describes doesn't show up until a few paragraphs later. 

If we can clear that up, the rest worked great!

Reply:
Good point! Adjusted with the specific version instructions.